Cargo.toml

```diff
@@ -141,5 +141,16 @@ pallet-hotfix-sufficients = { version = "1.0.0", path = "frame/hotfix-sufficient
 # Frontier Template
 frontier-template-runtime = { path = "template/runtime", default-features = false }
 #GGX
-account-filter = { package = "substrate-account-filter", git = "https://github.com/akorchyn/substrate-account-filter.git", default-features = false, branch = "polkadot-v0.9.35" }
-# pallet-collective = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.35", default-features = false }
\ No newline at end of file
+account-filter = { package = "substrate-account-filter", git = "https://github.com/GoldenGateGGX/substrate-account-filter", branch="polkadot-v0.9.35", default-features = false }
+#Astar
+pallet-xvm = { version = "0.2.1", path = "frame/pallet-xvm", default-features = false }
+pallet-evm-precompile-xvm = { version = "0.1.0", path = "precompiles/xvm", default-features = false }
+pallet-chain-extension-xvm = { version = "0.1.0", path = "chain_extensions/xvm", default-features = false }
```

Comment:
Rather than include these in our repo, is it possible to have the upstream pallets as a dependency here?

Reply:
I couldn't manage to do so initially, so I integrated it inside. Now, I will make one more iteration with a better understanding. Though, there are troubles with versioning. Astar pallets depend on its fork of frontier (I wonder if we want to use it.)

Maybe it would be easier if we forked astar-frame repo. Not sure.

Reply:
> Maybe it would be easier if we forked astar-frame repo. Not sure.

I'm inclined to have separate `ggx-frame` and `node` repos (as you have suggested out of band)...